src/pages/workspace/withPolicy.tsx

```diff
@@ -5,13 +5,15 @@ import type {ComponentType, ForwardedRef, RefAttributes} from 'react';
 import React, {forwardRef} from 'react';
 import type {OnyxEntry} from 'react-native-onyx';
 import {withOnyx} from 'react-native-onyx';
+import type {SettingsNavigatorParamList} from '@libs/Navigation/types';
 import policyMemberPropType from '@pages/policyMemberPropType';
 import * as Policy from '@userActions/Policy';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
+import type SCREENS from '@src/SCREENS';
 import type * as OnyxTypes from '@src/types/onyx';
 
-type PolicyRoute = RouteProp<{params: {policyID: string}}>;
+type PolicyRoute = RouteProp<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.BILLS | typeof SCREENS.WORKSPACE.INVOICES | typeof SCREENS.WORKSPACE.MEMBERS>;
```

Comment:
```suggestion
type PolicyRoute = RouteProp<SettingsNavigatorParamList, ValueOf<typeof SCREENS.WORKSPACE>>;
```

You can follow like I did in [this PR](https://github.com/Expensify/App/pull/34963/files#diff-a963b88bbf7285a2694c68ee310869f7a2c67fcdf1c5f82a14c9b210e782c447).

Reply:
Great advice, thanks!